source/content/guides/build-tools/10-update.md

```diff
@@ -1,21 +1,12 @@
 ---
 title: Build Tools
 subtitle: Update Your Project
-description: In step nine of the Build Tools guide, learn how to update your site as part of the continuous integration process.
+description: In step tengst of the Build Tools guide, learn how to update your site as part of the continuous integration process.
```

Comment:
"tengst"//"ten"

Reply:
the angst of not being able to spell numbers 😆

Reply:
Fixed 🤝